data/discovery/field/geographic-domain.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: 'n'
+datatype: curie
+field: geographic-domain
```

Comment:
This name feels too geeky, might be fine for alpha, but we need a common english name for the superset of country, territory and uk as it's a thing we'll use elsewhere.